steering_vectors/steering_vector.py

```diff
@@ -181,8 +194,13 @@ def hook_fn(_m: Any, _inputs: Any, outputs: Any) -> Any:
         delta = act
         if operator is not None:
             delta = operator(original_tensor, act)
-        mask = torch.ones(original_tensor.shape[1])
-        mask[:min_token_index] = 0
+        if isinstance(token_indices, Tensor):
+            mask = token_indices
+        elif token_indices is not None:
+            mask = torch.zeros(original_tensor.shape[1])
+            mask[token_indices] = 1
+        else:
```

Comment:
It looks like we don't need to handle the `token_indices is None` case, since above it gets initialized to `slice(None, None)`, if nothing is passed in. I think the type definition for this function can remove the `| None = None` part too.

Reply:
Done in [0343465](https://github.com/steering-vectors/steering-vectors/pull/21/commits/03434650e28b25533732909da51b682d03017a4c)